sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -1965,4 +1965,11 @@ class RapidsConf(conf: Map[String, String]) extends Logging {
     // user-provided value takes precedence, then look in defaults map
     conf.get(key).orElse(optimizerDefaults.get(key)).map(toDouble(_, key))
   }
+
+  /**
+   * To judge whether "key" is explicitly set by the users.
+   */
+  def isConfExplicitlySet(key: String): Boolean = {
+    shouldExplain && conf.contains(key)
```

Comment:
remove shouldExplain out of here though since this is now a generic function you could call for any configs.    We don't really need the check in this case.